vertx-web/src/main/java/io/vertx/ext/web/handler/sse/SSEHandlerImpl.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.vertx.ext.web.handler.sse;
+
+import io.vertx.core.Handler;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.sse.SSEConnection;
+import io.vertx.ext.web.handler.sse.SSEHandler;
+
+import java.util.List;
+
+public class SSEHandlerImpl implements SSEHandler {
+
+  private Handler<SSEConnection> connectHandler;
+
+  @Override
+  public synchronized void handle(RoutingContext context) {
+    HttpServerRequest request = context.request();
+    HttpServerResponse response = context.response();
+    List<String> acceptHeader = request.headers().getAll(HttpHeaders.ACCEPT.toString());
```

Comment:
Use the `context.parsedHeaders()` instead of handling the header yourself. This ensures order and quality are ensured. for example:

```java
List<MIMEHeader> acceptableTypes = context.parsedHeaders().accept();
if(!acceptableTypes.isEmpty()) {
  MIMEHeader selectedAccept = context.parsedHeaders()
    .findBestUserAcceptedIn(acceptableTypes, "text/event-stream");
  
  if (selectedAccept == null) {
    return 406;
...
```